packages/salesforcedx-vscode-soql/.eslintrc.json

```diff
@@ -44,7 +44,6 @@
     "no-var": "error",
     "jsdoc/no-types": "warn",
     "@typescript-eslint/semi": "warn",
-    "@typescript-eslint/class-name-casing": "warn",
```

Comment:
@lcampos I copied over the `eslintrc.json` from the [link](https://github.com/forcedotcom/source-deploy-retrieve/blob/develop/.eslintrc.json) you gave me. 
FYI, it looks like `class-name-casing` is deprecated: https://github.com/typescript-eslint/typescript-eslint/issues/1854#issuecomment-609913921